cli/dependabot-cmd.ts

```
@@ -70,23 +215,89 @@ export function dependabotCmd(sthis: SuperThis) {
+                : overallStatus === "FAILURE"
+                  ? "✗"
+                  : overallStatus === "PENDING"
+                    ? "○"
+                    : overallStatus === "REBASE-PENDING"
+                      ? "⚠️ "
+                      : overallStatus === "ERROR"
+                        ? "⚠"
+                        : "?";
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            console.log(
+              `  Status: ${statusEmoji} ${overallStatus} (${pr.statusCheckRollup!.length} check${pr.statusCheckRollup!.length !== 1 ? "s" : ""})`,
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)